Update doc (#281)
valory-xyz/mech: README.md

The project consists of three components:

-   Off-chain AI workers, each of which controls a Mech contract. Each AI worker is implemented as an autonomous service on the Autonolas stack.

> -  An on-chain protocol, which is used to generate a registry of AI Mechs, represented as NFTs on-chain.
> - An on-chain [MarketPlace](https://github.com/valory-xyz/ai-registry-mech/) which enable AI Mechs to easily deploy Mech contracts, relays service requests and deliveries to such Mech contracts, and guarantees service deliveries by implementing a reputation score and a take-over mechanism

Reviewer: ```suggestion
-   An on-chain protocol, which is used to generate a registry of AI Mechs, represented as NFTs on-chain.
-   An on-chain [MarketPlace](https://github.com/valory-xyz/ai-registry-mech/) which enable AI Mechs to easily deploy Mech contracts, relays service requests and deliveries to such Mech contracts, and guarantees service deliveries by implementing a reputation score and a take-over mechanism.
```

The project consists of three components:

> -   Off-chain AI workers, each of which controls a Mech. Each AI worker is implemented as an autonomous service on the Autonolas stack.

Reviewer 2: ```suggestion
-   Off-chain AI workers, each of which controls a Mech contract. Each AI worker is implemented as an autonomous service on the Autonolas stack.
```

> -   An on-chain protocol. This protocol consists of different types of smart contracts: Olas Mechs, which handle payments and receive requests for the off-chain part of the Mechs; the Mech Marketplace which relays requests to Mechs and guarantees quality to users by implementing a reputation score and a take-over mechanism. Mechs are services registered on the Olas registry which then register on the Mech Marketplace.  The repository for this protocol is: https://github.com/valory-xyz/ai-registry-mech. 

Reviewer 2: ```suggestion
-  An on-chain protocol, which is used to generate a registry of AI Mechs, represented as NFTs on-chain.
- An on-chain MarketPlace which enable AI Mechs to easily deploy Mech contracts, relays service requests and deliveries to such Mech contracts, and guarantees service deliveries by implementing a reputation score and a take-over mechanism
```

-   [Mech Hub](https://aimechs.autonolas.network/), a frontend which allows to interact with the protocol:
    -   Gives an overview of the AI workers in the registry.
    -   Allows Mech owners to create new workers.
    -   Allows users to request work from an existing worker.

> Note that Mechs which were deployed before the Mech Marketplace (called legacy Mechs) receive requests directly on their contract on-chain.

Reviewer 2: ```suggestion
_Note that Mechs which were deployed before the Mech Marketplace contracts (called legacy Mechs) receive and delivery services directly via their Mech contract._
```
